static/js/vue-cdr-access/tests/fixtures/displayWrapperFixtures.js

```diff
@@ -71,8 +71,6 @@ export const briefObjectData = {
                 }
             ],
             datastream: [
-                "thumbnail_small|image/png|fc77a9be-b49d-4f4e-b656-1644c9e964fc.png|png|6768|||",
-                "thumbnail_large|image/png|fc77a9be-b49d-4f4e-b656-1644c9e964fc.png|png|23535|||",
```

Comment:
maybe this should get a jp2 datastream since it's a collection that previously had thumbnails